### PR TITLE
[WIP] Test action woke.yml. [citest skip]

### PR DESCRIPTION
**Pull request by nhosoi**
Add a check for usage of terms and language that is considered non-inclusive. We are using the woke tool for this with a wordlist that can be found at https://github.com/linux-system-roles/tox-lsr/blob/main/src/tox_lsr/config_files/woke.yml

Apply "wokeignore:rule=" to the non-inclusive words.

### CODE REVIEW

**Comment by nhosoi** [comment]
[citest skip]

**Comment by nhosoi** [comment]
[citest skip]

**Comment by nhosoi** [comment]
[citest skip]

**Comment by nhosoi** [comment]
[citest skip]

**Comment by nhosoi** [comment]
@spetrosi, @richm, I'm testing `get-woke/woke-action@v0` in this pr. I'm having a difficulty in the error handling in woke-action. The issue is we'd like to interpret items set to `severity: warning` or `info` not an error in the test. But with the custom config file with the setting, it still ends up with an error...  

Here are the details.
I set `severity: warning` to the rule `sanity`.
https://github.com/nhosoi/tox-lsr/blob/woke/src/tox_lsr/config_files/woke.yml#L27

In `woke-action`, `sanity` is reported as a warning.
```
   Warning: `sanity` may be insensitive, use `confidence`, `quick check`, `coherence check` instead
```
But the action itself fails as follows:
```
https://github.com/linux-system-roles/logging/actions/runs/3650221774/jobs/6165933434
  Error: files with findings: 1
  2022-12-08T16:36:50Z FTL error="files with findings: 1"
  Error: Process completed with exit code 1.
```
I also tried disabling `fail-on-error`:
```
fail-on-error: false
```
Then, the action passes even if any real error is reported... (;_;)

**Comment by richm** [comment]
@nhosoi not sure - can you make it work if you run it locally?  You may have to dig into the source code to figure out what it is doing.  Or file an issue in the woke github - the authors are pretty responsive.

**Comment by nhosoi** [comment]
@richm, @spetrosi, could you please take a look at https://github.com/get-woke/woke/pull/252?
Thanks!

**Comment by richm** [comment]
> @richm, @spetrosi, could you please take a look at [get-woke/woke#252](https://github.com/get-woke/woke/pull/252)? Thanks!

lgtm

**Comment by nhosoi** [comment]
Thank you, @richm.

This PR 307 depends upon these PRs.
https://github.com/get-woke/woke/pull/252
https://github.com/linux-system-roles/tox-lsr/pull/97

Without https://github.com/get-woke/woke/pull/252 or its replacement, this woke action fails by the existence of the word `sanity`.

And once https://github.com/linux-system-roles/tox-lsr/pull/97 is merged, the following URL needs to be replaced with the official one.
https://github.com/nhosoi/logging/blob/woke-action/.github/workflows/woke.yml#L16

File: .github/workflows/woke.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# name: woke
+on:  # yamllint disable-line rule:truthy
+  - pull_request
+jobs:
+  woke:
+    name: woke
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```
Comment (richm) [inline .github/workflows/woke.yml:11]:
```suggestion
        uses: actions/checkout@v3
```

File: .github/workflows/woke.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# yamllint disable rule:line-length
+# name: woke
```
Comment (richm) [inline .github/workflows/woke.yml:2]:
```suggestion
name: woke
```

**Comment by nhosoi** [comment]
Closing this pr in favor of https://github.com/linux-system-roles/logging/pull/310.

**Pull request closed without merging**